4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-platforms-android-edge-to-edge-support.md

---
type: entry
section: platforms
subsection: android
rank: 3
importance: 4
anchor: edge-to-edge-support
title: Edge-to-edge support
text: |

>   Usually, developers had two choices for displaying content on the screen: either their app leaves the top and bottom areas for the system status and navigation bars, or their app could go full-screen, reclaiming all the space without displaying these bars.

Reviewer: ```suggestion
  Usually, developers had two choices for displaying content on the screen: either their app leaves the top and bottom areas for the system status and navigation bars, or their app could go full-screen, claiming all the space without displaying these bars.
```
Don't think "reclaiming" makes sense here, it didn't lose it before

>   Usually, developers had two choices for displaying content on the screen: either their app leaves the top and bottom areas for the system status and navigation bars, or their app could go full-screen, by reclaiming all the space forgoing displaying these bars.

Reviewer 2: ```suggestion
  Usually, developers had two choices for displaying content on the screen: either their app leaves the top and bottom areas for the system status and navigation bars, or their app could go full-screen, reclaiming all the space without displaying these bars.
```

>   From now on, once targetting SDK 35, there’s a new option. Android has a new feature called ["edge-to-edge"](https://developer.android.com/develop/ui/views/layout/edge-to-edge) display. It offers the developers the ability to draw on the entire screen, as if it was fullscreen, but with system bar overlays.

Reviewer 3: ```suggestion
  From now on, there’s a new export option. Android has a feature called ["edge-to-edge"](https://developer.android.com/develop/ui/views/layout/edge-to-edge) display. It offers the developers the ability to draw on the entire screen, as if it was fullscreen, but with system bar overlays.
```
It's available for all supported devices (Android 7+) and it's not a new feature, android 15 just enforces this now.

>   From now on, once targetting SDK 35, there’s a new option. Android has a new feature called ["edge-to-edge"](https://developer.android.com/develop/ui/views/layout/edge-to-edge) display. It pretty much merges the two options: it offers the developers the ability to draw on the entire screen, as if it was fullscreen, but with system bar overlays.

Reviewer 2: ```suggestion
  From now on, once targetting SDK 35, there’s a new option. Android has a new feature called ["edge-to-edge"](https://developer.android.com/develop/ui/views/layout/edge-to-edge) display. It offers the developers the ability to draw on the entire screen, as if it was fullscreen, but with system bar overlays.
```
I don't think it really "merges" the other two options - it's a third option, as already mentioned earlier in the paragraph.

  This gives your game a more modern look that matches Android’s design style.

> contributors:

Reviewer 4: @adamscott Can you also include @syntaxerror247 as contributor since he provided numerous bug fixes to get the feature stable.
